x-pack/osquerybeat/internal/action/action.go

```diff
@@ -17,8 +19,10 @@ var (
 )
 
 type Action struct {
-	Query      string
-	ID         string
+	Query string
+	ID    string
+	// The optional action timeout in seconds
```

Comment:
Is this comment true? I don't think it is, you get the timeout seconds from the map at L46 and then multiply it by `time.Second`, so this is in nanoseconds.

Reply:
yep, outdated comment will remove, was using seconds initially then switched to duration